readme.md

```diff
@@ -15,6 +15,7 @@ The AWS Cloud Development Kit (AWS CDK) is an open-source software development f
   * [Queue](#queue)
   * [CI/CD](#cicd)
   * [Monitoring](#monitoring)
+  * [Organization setup](#organization-setup)
```

Comment:
Should we name it just `Organizations` to be a bit more generic?
Please also add the new item to the end of the list - a general approach we follow

Reply:
* `Organizations` doesn't convey much meaning here (apart from we're using up AWS Organizations).  What about `Multi-accounts setup`. Would that make more sense? 
* Will move this to the end of the list.

Reply:
Have been pondering about this, I think `Multi-accounts setup` make most sense. People are more likely to search for this term. Put this in the new version.